packages/broker/src/StorageNodeRegistry.ts

```diff
@@ -41,7 +48,7 @@ export class StorageNodeRegistry {
     // TODO when we support multiple storage nodes, this method should actually return all
     // addresses. resends should try to fetch from random storage node, and if that fetch fails,
     // it should try to fetch from other addresses, too
-    private async getStorageNodeAddress(streamId: string): Promise<string|undefined> {
+    private async getStorageNodeAddress(streamId: string): Promise<string[]|undefined> {
```

Comment:
Rename to `getStorageNodeAddresses` and remove the comment. Could return an empty array if there are no storage nodes (then we don't need the special handling on `undefined` when iterating over results).

Reply:
Looks like I just said the same things, should've read your comment 1st lol.